src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -1105,5 +1105,20 @@ private static int OnSigChild(int reapAll, int configureConsole)
                 s_processStartLock.ExitWriteLock();
             }
         }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Gets
+        ///       the friendly name of the process.
+        ///    </para>
+        /// </devdoc>
```

Comment:
nit: maybe inline the short description.

```c#
        /// <devdoc>
        ///    <para>Gets the friendly name of the process.</para>
        /// </devdoc>
```

Reply:
Not sure how it ended up like that!